UpgradedVehicles/VehicleUpgrader.cs

```diff
@@ -161,20 +161,20 @@ private float EfficiencyPentalty(int speedBoosterCount)
                 return 1f;
 
             if (this.IsExosuit)
-                return 1.15f * speedBoosterCount;
+                return 1f + (0.15f * speedBoosterCount);
 
             if (this.IsSeamoth)
-                return 1.10f * speedBoosterCount;
+                return 1f + (0.10f * speedBoosterCount);
 
-            return 1f * speedBoosterCount;
+            return 1f + (0.125f * speedBoosterCount);
         }
 
         private float EfficiencyBonus(int powerModuleCount)
         {
             if (powerModuleCount == 0)
                 return 1f;
 
-            return 1f + powerModuleCount;
+            return 1f + (0.15f * powerModuleCount);
```

Comment:
This breaks the original design of how the engine efficiency modules were intended affect the power rating of the small vehicles.  

Again, the wiki is wrong. The original code does actually provide a full 100% power rating increase to the vehicle for each (stacking) engine efficiency module.  
This change here would be a nerf so hard that it would make the module practically useless.